code/debris/debris.cpp

```diff
@@ -79,6 +79,42 @@ static void debris_start_death_roll(object *debris_obj, debris *debris_p)
     debris_obj->flags.set(Object::Object_Flags::Should_be_dead);
 }
 
+int clearDebrisAction(sexp::SEXPParameterExtractor* extractor) {
+	bool hullDebris = extractor->getOptionalArg(true);
+	bool nonHullDebris = extractor->getOptionalArg(true);
+
+	for (auto& debris_entry : Debris) {
+		if ( !(debris_entry.flags[Debris_Flags::Used]) )
+			continue;
+
+		if (debris_entry.is_hull) {
+			if (hullDebris) {
+				debris_start_death_roll(&Objects[debris_entry.objnum], &debris_entry);
+			}
+		} else {
+			if (nonHullDebris) {
+				debris_start_death_roll(&Objects[debris_entry.objnum], &debris_entry);
+			}
+		}
+	}
+
+	return 0;
```

Comment:
0 is equivalent to `SEXP_FALSE`, but all "void-ish" sexp operators return `SEXP_TRUE`, or 1.

Reply:
This value gets returned directly to the SEXP system so that is very relevant. Changed it to `SEXP_TRUE`, thanks for the hint.